src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-transform: uppercase;
```

Comment:
text-transform should be on elements which we want to transform, not for the whole body, please move it to nav__link